### PR TITLE
Added ability to hide hydra hint on show

### PR DESCRIPTION
**Pull request by miversen33**
Implementation of #24 

I am not completely sure of this as a feature. I personally like it, but I don't know that it should be added to hydra. Unfortunately there doesn't seem to be a better way to handle this.

Thoughts/opinions?

### CODE REVIEW

**Comment by benlubas** [comment]
Why only the pink hydra?

**Comment by miversen33** [comment]
I suppose I could add this to "non-pink" hydra's but in my head the idea was, pink is the only hydra that can be "backgrounded" (IE, you are allowed to press other keys after the hydra body). It didn't make much sense to me to allow hiding the other hydras but it isn't a huge deal if you feel we should

**Comment by benlubas** [comment]
Yeah I think we might as well. 

My first thought is for the window Hydra you might not want it to block your view of windows but still be able to pull up the hint.

**Comment by miversen33** [comment]
Sounds valid to me. I will get that added to the PR this morning

**Comment by miversen33** [comment]
Added support for other hydras as well. Give it a try and lemme know

**Comment by miversen33** [comment]
Don't merge this yet, I found a bug with toggling the hint multiple times

**Comment by miversen33** [comment]
Ok I _think_ its stable now

File: README.md
Comment (benlubas) [inline README.md:264]: I think this should be `Hydra.hint:show()`, and maybe we should mention `Hydra.hind:close()` as well.

**Review by benlubas — CHANGES_REQUESTED** [review]
Looks good to me. Tested it quickly and it seems to work well, just a documentation change and then we can merge it.

File: README.md
Comment (miversen33) [reply README.md]: I agree. Done

**Review by miversen33 — COMMENTED** [review]
Lol this breaks func processing on non-pink hydras it seems 🙃

**Comment by miversen33** [comment]
Fixed again lol. I almost want to say we need unit tests but I don't even know how we would approach setting those up for this project.

**Review by benlubas — APPROVED** [review]
lgtm, I'll let you merge in case there are other issues

**Comment by miversen33** [comment]
I'm dicking with my original idea today so I will finish that out and if I see no more issues popup, I will merge. I apologize for that, I should have more thoroughly tested this out before submitting the PR

**Comment by miversen33** [comment]
All seems well. Merging and closing

**Pull request merged**
